modules/logging-loki-storage-azure.adoc

```diff
@@ -24,3 +24,34 @@ $ oc create secret generic logging-loki-azure \
   --from-literal=account_key="<azure_account_key>"
 ----
 <1> Supported environment values are `AzureGlobal`, `AzureChinaCloud`, `AzureGermanCloud`, or `AzureUSGovernment`.
+
+== Azure storage for STS enabled clusters
+
+You can create a secret manually by running the following command:
+[source,terminal,subs="+quotes"]
+----
+oc -n openshift-logging create secret generic logging-loki-azure-cloud-credentials
+--from-literal=environment="AzureGlobal"
+--from-literal=account_name="<storage_account_name>"
+--from-literal=container="<container_name>"
+----
+
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using Azure AD Workload Identity. Secrets created by using CCO utility (`ccoctl`) use different format to authenticate.
+
+.Example Azure AD Workload Identity secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: <target-namespace>
+  name: <target-secret-name>
+data:
+  azure_client_id: <client_id>
+  azure_federated_token_file: </path/to/mounted/service/account/token> # <1>
+  azure_region: <region>
+  azure_subscription_id: <subscription_id>
+  azure_tenant_id: <tenant_id>
+----
+<1> Refreshed hourly.
```

Comment:
Similar to the comment on the CCO secret generated with AWS, I don't think this is relevant information for the user.